app/models/manageiq/providers/terraform_enterprise/automation_manager/orchestration_stack/status.rb

```diff
@@ -0,0 +1,52 @@
+class ManageIQ::Providers::TerraformEnterprise::AutomationManager::OrchestrationStack::Status < ::OrchestrationStack::Status
+  def initialize(status)
+    self.status = status
+    self.reason = self.class.run_state_to_description(status)
```

Comment:
TFE doesn't return a "human readable" description of the status but they have a full list statuses in the API docs.